### PR TITLE
Docker compose image

### PR DESCRIPTION
**Pull request by mateusduboli**
Create a simple docker-compose recipe to lift a redis machine for local testing.

### CODE REVIEW

**Comment by mateusduboli** [comment]
This PR depends on #2 , since now `Dockerfile` is based on the `Makefile`

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	go clean
+
+env:
+	export GOPATH=${GOPATH}:${PWD}
+
+docker:
+	docker-compose rm
+	docker-compose build
+	docker-compose up
+	docker-compose up
```
Comment (thiagocaiubi) [inline Makefile]: WTF?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	redis-metrics
+
+build: dependencies
+	go install github.com/daime/redis-metrics
+
+dependencies:
+	go get github.com/garyburd/redigo/redis
+
+test:
+	go test
```
Comment (thiagocaiubi) [inline Makefile]: go test -v ./...

**Comment by thiagocaiubi** [comment]
LGTM

**Pull request merged**